Ginger/Ginger/UserControlsLib/ImageMakerLib/ImageMakerControl.xaml.cs

```diff
@@ -521,6 +524,18 @@ private void SetImage()
                 case eImageType.Camera:
                     SetAsFontAwesomeIcon(FontAwesomeIcon.Camera);
                     break;
+                case eImageType.ExcelFile:
+                    SetAsFontAwesomeIcon(FontAwesomeIcon.FileExcelOutline);
+                    break;
+                case eImageType.PlusSquare:
+                    SetAsFontAwesomeIcon(FontAwesomeIcon.PlusSquare);
+                    break;
+                case eImageType.Th:
+                    SetAsFontAwesomeIcon(FontAwesomeIcon.ThList);
+                    break;
+                case eImageType.ThLarge:
+                    SetAsFontAwesomeIcon(FontAwesomeIcon.ThLarge);
+                    break;
                 #endregion
```

Comment:
as mentioned in preivous reviews, delete the unused eImageTypes and for needed one they need to have logic name of what they used for and not what shown

Reply:
removed now